packages/patternfly-3/patternfly-react/less/expand-collapse.less

```diff
@@ -0,0 +1,15 @@
+.expand-collapse-pf {
+  .expand-collapse-pf-left {
+    text-align: left;
+  }
+
+  .expand-collapse-pf-center {
+    text-align: center;
+  }
+
+  .btn.btn-link {
+    &:focus {
+      outline: none;
+    }
+  }
+}
```

Comment:
This file needs to be updated to what changed in the sass version.

Reply:
done